src/main/java/org/carlmontrobotics/commands/AutoMATICALLYGetNote.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.carlmontrobotics.commands;
+
+import org.carlmontrobotics.Constants.*;
+import org.carlmontrobotics.subsystems.*;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.ProxyCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class AutoMATICALLYGetNote extends Command {
+  /** Creates a new AutoMATICALLYGetNote. */
+  private Drivetrain dt;
+  private IntakeShooter effector;
+  private Limelight limelight;
+
+  public AutoMATICALLYGetNote(Drivetrain dt, IntakeShooter effector) {
+    addRequirements(this.dt = dt);
+    addRequirements(this.effector = effector);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  @Override
+  public void initialize() {
+    //new Intake().finallyDo(()->{this.end(false);});
+    dt.setFieldOriented(false);
+  }
+
+  @Override
+  public void execute() {
+    double radErr = Units.degreesToRadians(LimelightHelpers.getTX(Limelightc.INTAKE_LL_NAME));
+    double distErr = limelight.getDistanceToNote(); //meters
+    dt.drive(Math.max(distErr*2, .5),0,Math.max(radErr*2,.5));
```

Comment:
If note is not detected do you want the drivetrain to move at all?

Reply:
Also is the "2" some form of kP? Please use constants to make it more clear what they are. I suggest reusing existing constants such as the `double[] xPIDController` in `Constants.java` (You might be able to just combine x and y pid controller together honestly)